syntax = "proto2";
option optimize_for = LITE_RUNTIME;
import "world_landmass.proto";

package legends;

message world_landmass_request {
}
message world_landmass_response {
  repeated dfproto.world_landmass all = 1;
}

service legends_service {
  rpc get_world_landmass(world_landmass_request) returns(world_landmass_response);
}
